src/streaming-ingestion/lambdas/custom-resource/create-schemas.ts

```diff
@@ -0,0 +1,361 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { RedshiftDataClient } from '@aws-sdk/client-redshift-data';
+import { CdkCustomResourceHandler, CdkCustomResourceEvent, CdkCustomResourceResponse, CloudFormationCustomResourceEvent, Context, CloudFormationCustomResourceUpdateEvent } from 'aws-lambda';
+import { getRedshiftClient, executeStatementsWithWait } from '../../../analytics/lambdas/redshift-data';
+import { getSqlContent, getSqlContents } from '../../../analytics/private/utils';
+import { logger } from '../../../common/powertools';
+import { CreateStreamingIngestionSchemas, MustacheParamType } from '../../redshift/model';
+
+export type ResourcePropertiesType = CreateStreamingIngestionSchemas & {
+  readonly ServiceToken: string;
+}
+
+export const handler: CdkCustomResourceHandler = async (event: CloudFormationCustomResourceEvent, context: Context) => {
```

Comment:
What's the difference between this one and the one in data modeling(Redshift)? Could we create a base class for sharing the logic to creating schemas in Redshift?

Reply:
Added `createSchemasInRedshift` function in `redshift-data.ts` shared for use by two modules.